packages/back-end/src/controllers/metrics.ts

```diff
@@ -450,9 +451,19 @@ export async function postMetrics(
     userIdColumns,
     userIdTypes,
     templateVariables,
+    managedBy,
   } = req.body;
 
-  if (!context.permissions.canCreateMetric({ projects })) {
+  if (
+    managedBy === "admin" &&
+    !context.hasPremiumFeature("manage-official-resources")
+  ) {
+    throw new Error(
+      "Your organization's plan does not support creating official metrics.",
+    );
+  }
+
+  if (!context.permissions.canCreateMetric(req.body)) {
```

Comment:
Nit: Since you added these checks in the model, you can remove all of this here.